taworks/taform/static/taform/css/style.css

```diff
@@ -79,6 +79,10 @@ table {
 .tg .tg-yw4l {
     vertical-align: top
     }
+.tg .test{
```

Comment:
remove if we aren't using it. We should try to make better use of better naming for future styling